src/clj/clojure/main.clje

```diff
@@ -311,7 +311,7 @@ by default when a new command-line REPL is started."} repl-requires
 (defn- initialize
   "Common initialize routine for repl, script, and null opts"
   [args inits]
-  (ns 'clje.user)
+  (in-ns 'clje.user)
```

Comment:
This used to be `ns` to ensure `clojure.core` was referred, but after this change the `clje.user` namespace always exists.